src/Configuration/Config.Json/src/Microsoft.Extensions.Configuration.Json.csproj

```diff
@@ -5,6 +5,10 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageTags>$(PackageTags);json</PackageTags>
     <IsShipping>true</IsShipping>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsPackable>true</IsPackable>
+    <NoWarn>$(NoWarn);3021</NoWarn>
+    <LangVersion>latest</LangVersion> 
```

Comment:
```suggestion
```
You can remove this too. The repo-wide default was update to 7.3 a few commits ago.